Pod/Classes/LocationManager.swift

```diff
@@ -10,244 +10,278 @@
 import CoreLocation
 import PromiseKit
 
-public enum LocationManagerError: ErrorType {
-    case LocationServiceDisabled
-    case CannotFetchLocation
+public enum LocationManagerError: Error {
+    case locationServiceDisabled
+    case cannotFetchLocation
 }
 
-public enum LocationManagerAuthorizationError: ErrorType {
-    case KeyInPlistMissing
+public enum LocationManagerAuthorizationError: Error {
+    case keyInPlistMissing
 }
 
-public class LocationManager: NSObject, CLLocationManagerDelegate {
-    
-    public static let locationDidChangeAuthorizationStatusNotification = "locationDidChangeAuthorizationStatusNotification"
-    
-    public static let sharedManager = LocationManager()
+open class LocationManager: NSObject, CLLocationManagerDelegate {
+
+    typealias AuthorizationFulfillment = (CLAuthorizationStatus) -> Void
```

Comment:
I believe we use US spelling, like all the system frameworks. Therefore it is okay.